titles/getting-started-rhdh/title-getting-started.adoc

```diff
@@ -19,7 +19,6 @@ include::modules/installation/ref-rhdh-sizing.adoc[leveloffset=+1]
 // supported configs and customization
 include::modules/getting-started/ref-rhdh-supported-configs.adoc[leveloffset=+1]
 include::assemblies/assembly-add-custom-app-file-openshift.adoc[leveloffset=+2]
-include::modules/getting-started/proc-add-source-control-rhdh-catalog.adoc[leveloffset=+2]
```

Comment:
In the [Bulk Import PR](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/616), wasn't the Bulk Import section added under Source Control? I noticed that in this PR, we're removing it. Just trying to clarify as it seems the structure and preview haven't been generated.

Reply:
Indeed, I updated this other PR to reflect the change.